app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
```

Comment:
Let's remove the pages controller if we are not implementing it in this story.

Reply:
Doesn't it make sense to have a pages controller to control the static pages, like home for example?

Reply:
what is the view of this page ? can you share a Screenshot?

Reply:
<img width="1701" alt="image" src="https://github.com/tgi-wtag/Journey-tg/assets/141820846/cc6e30bc-219b-4098-95a3-4b1dcd3ef663">

Reply:
ok, my idea was to have one controller for the static pages, which can handle the 'home', 'about' or other pages which don't need an own controller